docs/usage/training_time_compression/quantization_aware_training/Usage.md

```diff
@@ -75,7 +75,7 @@ ov_quantized_model = ov.convert_model(stripped_model)
 
 The complete information about compression is defined by a compressed model and a NNCF config.
 The model characterizes the weights and topology of the network. The NNCF config - how to restore additional modules introduced by NNCF.
-The NNCF config can be obtained by `quantized_model.nncf.get_config()` on saving and passed to the
+The NNCF config can be obtained by `nncf.torch.get_config(quantized_model)` on saving and passed to the
```

Comment:
As you mentioned earlier, it won't work
https://github.com/openvinotoolkit/nncf/pull/3359/files#r2005207359

Reply:
updated usage